pkg/capability/capability.go

```diff
@@ -0,0 +1,135 @@
+package capability
+
+import (
+	"sort"
+
+	configv1 "github.com/openshift/api/config/v1"
+)
+
+type ClusterCapabilities struct {
+	KnownCapabilities             map[configv1.ClusterVersionCapability]struct{}
```

Comment:
if the type is called `ClusterCapabilities` I believe we do not need to repeat the word in its members, and call the members just `Known`, `Enabled` and `ImplicitlyEnabled`